lib/projects.js

```diff
@@ -41,6 +41,10 @@ var projects = {
   "webpack-cli": {
     name: 'webpack-cli',
     q: 'repo:webpack/webpack-cli is:open is:issue label:"Good First Contribution"'
+  },
+  jest: {
+    name: 'jest',
```

Comment:
Same goes for the Jest website, using "Jest" rather than the lower-case "jest". 


```suggestion
    name: 'jest',
```